windows/installer-source/KaliteSetupScript.iss

```diff
@@ -342,7 +342,7 @@ var
 
 begin
     PipPath := 'C:\Python27\Scripts\pip.exe';
-    ﻿PipCommand := 'install ' + '"' + ExpandConstant('{app}\ka-lite\dist\ka-lite-static-')  + '{#MyVersion}' + '.zip' + '"';
+    PipCommand := 'install ' + '"' + ExpandConstant('{app}\ka-lite\dist\ka-lite-static-')  + '{#MyVersion}' + '.zip' + '"';

```

Comment:
Also, `{#MyVersion}` returns a different version string than the output of `setup.py sdist --static` (namely `0.14alpha1` vs `0.14a1`). Let me try to resolve that in this PR as well.